README.md

```diff
@@ -5,5 +5,6 @@ Design document: https://docs.google.com/document/d/12gZow_JTJjRrmaD2mNTmYniLhyx
 
 [![build-and-test](https://github.com/celestiaorg/optimint/actions/workflows/test.yml/badge.svg)](https://github.com/celestiaorg/optimint/actions/workflows/test.yml)
 [![golangci-lint](https://github.com/celestiaorg/optimint/actions/workflows/lint.yml/badge.svg)](https://github.com/celestiaorg/optimint/actions/workflows/lint.yml)
+[![Go Report Card](https://goreportcard.com/badge/github.com/celestiaorg/optimint)](https://goreportcard.com/report/github.com/celestiaorg/optimint)
```

Comment:
Nice, while at it another thing we should start adding (here and in other repos) is a twitter badge. See for example https://github.com/tharsis/ethermint#-ethermint-

Reply:
Great idea! Just added :arrow_down: